### ISSUE TITLE
(Talkback): Menu in Stock Camera app not accessible with Talkback

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#328]
```
What steps will reproduce the problem?
1. Open the Camera app.
2. Flick to the last option or use bottom right corner to access the 'menu' 
button.
3. Visually, a menu is shown with options to select what camera to use, HDR 
settings etc. Talkback does not see these options.

What is the expected output? What do you see instead?
Expected is that I can modify these options using the Talkback screenreader. 
However, Talkback appears to be unaware of these menu options, which makes it 
impossible for me to switch to front-facing camera.

What version of the product are you using? On what operating system?
Google nexus 4 on Android 4.3. Talkback v3.5 latest beta.

Please provide any additional information below.
I have not found a 3rd party app yet that can serve as a replacement for the 
Camera app. Therefore, this functionality appears completely unavailable to 
blind Android users.

```

Original issue reported on code.google.com by `florianb...@gmail.com` on 3 Nov 2013 at 9:26

**Comment by GoogleCodeExporter** [comment]
```
From TalkBack itself, there's not much we can do to improve the accessibility 
of this menu since it's an entirely custom control.

I have filed a bug report with the Camera app.
```

Original comment by `caseybur...@google.com` on 5 Dec 2013 at 6:16
- Changed state: **Done**
